introduction.md

```diff
@@ -0,0 +1,7 @@
+I started in the Ruby world, but now I love JavaScript.
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?